### ISSUE TITLE
Add credit for Infinity image for redesigned homepage

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3342]
### Overview
We need to add a credit for a new image used in the redesigned version of the homepage so that the [Credits page](https://www.hackforla.org/credits/) is up to date.

### Details
From the [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724), here is a screenshot of the credit's information for reference:
  <details>
    <summary>Click here to see the screenshot of the Infinity credit in Figma</summary>
    <img alt="" src="https://user-images.githubusercontent.com/31293603/178128070-e79079e0-9771-4cc6-afcd-16b533eb0333.png">
  </details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] In `_data/internal/credits` directory, create a file called `infinity.yml`
- [x] In the file you just created, copy and paste the following:
```
---
# The title from the source provider
title: Infinity
# The url where you can get direct access to the image on the provider site
title-link: https://thenounproject.com/search/?q=infinity&i=686117
# The type of content - image, audio, or video 
content-type: image
# The page it's used on
used-in: Homepage
# Name of the artist
artist: Yeong Rong Kim
# Name of the image database you found it on
provider: Noun Project
# Please give the general site URL, not the link to the image
provider-link: https://thenounproject.com/
# Location in our GitHub repo
image-url: /assets/images/hero/infinite.svg
# Alt text for the image on the Credits page
alt: ''
---
```
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3130 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue #3130 to the New Issue Approval column and remove the Dependency label
  
### Resources/Instructions
- This issue is tracked in the epic #3130.
- [Credits page](https://www.hackforla.org/credits/)
- [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Credits yml file template for Iconography and Imagery](https://github.com/hackforla/website/wiki/Credits-yml-file-template-for-Iconography-and-Imagery)
- [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)

**Comment by JessicaLucindaCheng** [comment]
This issue would usually have the milestone 01. Compliance. However, it's specifically for the homepage launch, which has not happened yet.

**Comment by arpitapandya** [comment]
Availability for this week: 6hrs
My estimated ETA for completing this issue: 2hrs

**Comment by arpitapandya** [comment]
Time spent: 6hrs to complete
Availability: 6hrs for next week
Blockers: None for this issue
